js/data/bucket.js

```diff
@@ -79,7 +79,7 @@ class Bucket {
         let type = options.layer.type;
         if (type === 'fill' && (!options.layer.isPaintValueFeatureConstant('fill-extrude-height') ||
             !options.layer.isPaintValueZoomConstant('fill-extrude-height') ||
-            options.layer.getPaintValue('fill-extrude-height') !== 0)) {
+            options.layer.getPaintValue('fill-extrude-height', {}) !== 0)) {
```

Comment:
Would we head off similar bugs if we added this default value further upstream? (such as within `StyleLayer#getPaintValue`)